src/components/Form/ExitFormDialogProvider.tsx

```diff
@@ -237,8 +238,27 @@ const ExitFormDialogProvider = ({ children }) => {
     shouldBlockNavigation,
     setEnableExitDialog,
     setExitDialogSubmitRef: setSubmitRef,
-    setIsSubmitting
+    setIsSubmitting,
+    submit: handleSubmit
```

Comment:
Since atm we should avoid using this outside of tests maybe we could add a comment?

Reply:
Fair enough